_includes/overview/provisioning.html

```diff
@@ -1,10 +1,5 @@
-<p>
-  For provisioning purposes you'll want to search for hosts that are unallocated. Do not allocate hosts in any other state.
-</p>
-<p>
-  To provision a host just click the 'Provision Server' link in the actions drop down. You will be presented with a dialog
-  like the following:
-</p>
+<p>For provisioning purposes you'll want to search for hosts that are unallocated (or sometimes maintenance), as they are not already claimed and in production. Do not provision hosts in any other state!</p>
+<p>To provision a host just click the 'Provision Server' link in the actions drop down. Assets in Allocated are not able to be provisioned until you move them into maintenance or unallocated. You will be presented with a dialog like the following:</p>
```

Comment:
To provision a host use the provisioning api, POST /api/provision/:tag, or in the web UI ...

please split these lines too